TODO.txt

```diff
@@ -82,10 +82,6 @@ Other (2021)
 * Monitor when multibuild devel gets merged into master for python 3.8
   compatibility https://github.com/matthew-brett/multibuild/issues/284
   and update osx_install to point to the correct branch
-* When ``numpy`` is set to >= 1.16, remove the warning assertion in
-  ``skimage/exposure/tests/test_exposure.py::test_rescale_nan_warning``
-  regarding ``invalid value encountered in reduce``.
-  regarding ``Passing `np.nan` to mean no clipping in np.clip``.
```

Comment:
I think this instruction was incorrect. The warning did not occur on old NumPy, but does on 1.17+. I pushed a commit to restore the warning, but make it non-optional now.

Reply:
Nevermind, this does not actually appear to warn in my local test environment. I cancelled CI here for now and will investigate further...

Reply:
Hopefully CI will pass now. It turns out that only some of the parameterized test cases try to clip using NaN values

Reply:
ah ok! Thanks for working on the PR btw, really cool!